build/charts/antrea/values.yaml

```diff
@@ -1,6 +1,13 @@
 # -- Container image to use for Antrea components.
```

Comment:
You are still missing a deprecation notice here. For example:
```yaml
# -- Container image to use for Antrea components.
# DEPRECATED: use agentImage and controllerImage instead
image: {}
```

Reply:
will add it.